### ISSUE TITLE
Prefetch only links with content.json file available

### ISSUE THREAD

**Issue by synalysis** [dillonkearns/elm-pages#37]
Currently all links going to the same site are prefetched which prevents links to downloadable files to work as well as <iframe> elements injected in Markdown files.

**Comment by dillonkearns** [comment]
@synalysis would you mind trying out this branch to see if it solves the problem you were experiencing?

You can just git clone this repo, then

```shell
# cd into the repo
git checkout dont-preload-unknown-paths
npm install -g .
```

Would appreciate your insight on this!

**Comment by synalysis** [comment]
Works as designed - both <iframe> and download file 💪

**Comment by synalysis** [comment]
I noticed that when I open a subpage directly (or reload it), the start page is visible for a moment before the correct page is displayed. Don't know if this is a new issue.

It is not noticeable with the dev server but when installed on a web server. Both in Safari and Firefox the blinking start page can be observed: [https://hexinator.com/order](https://hexinator.com/order)

**Comment by dillonkearns** [comment]
Fantastic, thank you for checking @synalysis! 🙏

That issue you're seeing is high on my list, this is the problem there: https://github.com/dillonkearns/elm-pages/issues/52.

I'll get this released shortly 👍

**Comment by dillonkearns** [comment]
This has been released with NPM package 1.2.3 and Elm package 3.0.1 👍
